client/my-sites/domains/domain-search/index.jsx

```diff
@@ -13,13 +13,14 @@ import { withShoppingCart } from '@automattic/shopping-cart';
 /**
  * Internal dependencies
  */
+import config from '@automattic/calypso-config';
```

Comment:
1000% nit: is this not technically an external dependency...?

Reply:
Frankly, I don't know. It used to be but now I'm not sure. I've checked a few other files, and sometimes it's internal and sometimes it's not.